composer.json

```diff
@@ -14,21 +14,31 @@
   ],
   "require": {
     "php": ">=7.1",
-    "fond-of-spryker/company-type": "dev-master"
+    "fond-of-spryker/company": "dev-master",
+    "fond-of-spryker/company-business-unit": "dev-master",
+    "fond-of-spryker/company-type": "dev-master",
+    "spryker/company-role": "^1.0.0"
```

Comment:
@SpiderDan98 did you consider all possible versions?

Reply:
@daniel-rose Yes now